x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -20,8 +21,8 @@ export const installTemplates = async (
   // install any pre-built index template assets,
   // atm, this is only the base package's global index templates
   // Install component templates first, as they are used by the index templates
-  installPreBuiltComponentTemplates(pkgName, pkgVersion, callCluster);
-  installPreBuiltTemplates(pkgName, pkgVersion, callCluster);
+  await installPreBuiltComponentTemplates(pkgName, pkgVersion, callCluster);
```

Comment:
Can we do this two operation concurently inside a `Promise.all`

Reply:
The second depends on the first being completed, so we can't do them concurrently.